Documentation/admin/ingress.md

```diff
@@ -150,8 +150,16 @@ metadata:
 
 When rules are defined and the Ingress Controller is running, traffic to your Service will be routed appropriately.
 
+### HTTP header limitations
+
+Using reverse proxies that limit HTTP headers to 4K may result in errors for large group counts. If using HTTP headers for large cookie replacement, set the 
+`large-client-header-buffers` flag to `4 64k` on the Ingress Configmap `tectonic-custom-error` to increase the limit in Ingress to EG 64k.
```

Comment:
s/EG/e.g.,/

- Consistency of K vs k for kilobytes
- The effect of this setting is the allocation of _4_ buffers of 64 kilobytes each

Reply:
ah. that makes way more sense.